backend/kernelCI_app/views/treeCommitsHistory.py

```diff
@@ -73,55 +184,67 @@ def get(self, request, commit_hash):
                 builds AS b
                 ON c.id = b.checkout_id
             WHERE
-                c.git_repository_branch = %(git_branch_param)s
-                AND c.git_repository_url = %(git_url_param)s
-                AND c.origin = %(origin_param)s
+                {build_counts_where}
+            GROUP BY
+                c.git_commit_hash
+        ),
+        boots_counts AS (
+            SELECT
+                c.git_commit_hash,
+                SUM(CASE WHEN t.status = 'FAIL' AND t.path LIKE 'boot%%' THEN 1 ELSE 0 END)
+                AS boots_fail_count,
+                SUM(CASE WHEN t.status = 'ERROR' AND t.path LIKE 'boot%%' THEN 1 ELSE 0 END)
+                AS boots_error_count,
+                SUM(CASE WHEN t.status = 'MISS' AND t.path LIKE 'boot%%' THEN 1 ELSE 0 END)
+                AS boots_miss_count,
+                SUM(CASE WHEN t.status = 'PASS' AND t.path LIKE 'boot%%' THEN 1 ELSE 0 END)
+                AS boots_pass_count,
+                SUM(CASE WHEN t.status = 'DONE' AND t.path LIKE 'boot%%' THEN 1 ELSE 0 END)
+                AS boots_done_count,
+                SUM(CASE WHEN t.status = 'SKIP' AND t.path LIKE 'boot%%' THEN 1 ELSE 0 END)
+                AS boots_skip_count
+            FROM
+                relevant_checkouts AS c
+            INNER JOIN
+                builds AS b
+                ON
+                c.id = b.checkout_id
+            LEFT JOIN
+                tests AS t
```

Comment:
this query can gets very heavy after they turn back the notifications in the backend, I think would be smart to create a CTE with tests so that join is made only a single time